tests/commands/gen-ios/__snapshots__/SEGTestTrackingPlanAnalytics.m

```diff
@@ -217,10 +223,18 @@ + (SEGExampleEvent *)initWithBlock:(SEGExampleEventBuilderBlock)block
         @throw [NSException exceptionWithName:@"Missing Required Property" reason:@"SEGExampleEvent is missing a required property: requiredInt" userInfo:NULL];
     }
 
+    if (builder.requiredNullableString == NULL) {
+        @throw [NSException exceptionWithName:@"Missing Required Property" reason:@"SEGExampleEvent is missing a required property: requiredNullableString" userInfo:NULL];
+    }
```

Comment:
Note: nullable types are now supported (as in, the client will compile), but this runtime validation is (clearly) not the desired behavior. However, this will change when we add in a full JSON Schema library to replace these required property checks, so I'm not going to bother with changing this behavior.